app/routes/_index.tsx

```diff
@@ -1,47 +1,56 @@
 import { type MetaFunction } from "@remix-run/node";
+import WeAreBlock from "~/components/info";
+import MissionStatement from "~/components/landing";
+
 export const meta: MetaFunction = () => {
   return [
-    { title: "New Remix App" },
-    { name: "description", content: "Welcome to Remix!" },
+    { title: "Code4Community" },
+    { name: "description", content: "Welcome to Code4Community!" },
   ];
 };
 
 export default function Index() {
+  const c4cPurple = '#605ACD';
+  const c4cLightPurple = '#F8EDFF';
```

Comment:
these are never used, you should setup the eslint extension in vscode or whatever ide you use.
I'll add a check to our precommit hook so it's visible before you commit in the future as well.